src/redefined.resolver.ts

```diff
@@ -33,7 +35,16 @@ export class RedefinedResolver {
 
     async resolve(domain: string, networks?: string[], options?: CustomResolverServiceOptions): Promise<ResolverResponse> {
         if (!this.resolvers.length) {
-            this.resolvers = await RedefinedResolver.configResolvers();
+            if (!this.config) {
+                this.config = new Promise((resolve) => {
+                    try {
+                        resolve(fetch(`${config.CONFIGS_URL}?v=${new Date().valueOf()}`).then(n => n.json()));
```

Comment:
Btw you can still use ES6 with `await`

If you want to use this construct, it would be better to change to this entry
```
fetch(`${config.CONFIGS_URL}?v=${new Date().valueOf()}`)
    .then(n => resolver(n.json()))
```

Also i'm not sure if this `catch` works with this construct
Need to use `async``await` this this `try {} catch() {}`
or
```
fetch()
  .then()
  .catch(e => {
     console.error(e)
   })
```